.gitignore

```diff
@@ -9,3 +9,7 @@ target/
 .settings/
 .vscode
 bin/
+
+#gradle
```

Comment:
Is this a debug leftover?

Reply:
no, not a debug leftover. A comment on "here start the gradle-based gitignore entries"

Reply:
kind of a section header, as with "#vscode" above.

Reply:
Ah. Then at least for me it would help if you add a space in between the `#` and the header. Then it does not look like the value is commented out.